cli/cmd/ps.go

```diff
@@ -59,50 +54,69 @@ func PsCommand() *cobra.Command {
 	cmd.Flags().BoolVarP(&opts.quiet, "quiet", "q", false, "Only display IDs")
 	cmd.Flags().BoolVarP(&opts.all, "all", "a", false, "Show all containers (default shows just running)")
 	cmd.Flags().BoolVar(&opts.json, "json", false, "Format output as JSON")
+	cmd.Flags().StringVar(&opts.format, "format", "", "Format the output. Values: [pretty | json | go template]. (Default: pretty)")
+	_ = cmd.Flags().MarkHidden("json")
```

Comment:
Maybe a comment mentionning this is used in VSCode Docker extension for the moment

Reply:
Done